ironfish/src/fileStores/config.ts

```diff
@@ -304,6 +305,7 @@ export class Config extends KeyStore<ConfigOptions> {
       rpcTcpSecure: false,
       tlsKeyPath: files.resolve(files.join(dataDir, 'certs', 'node-key.pem')),
       tlsCertPath: files.resolve(files.join(dataDir, 'certs', 'node-cert.pem')),
+      rpcAuthTokenPath: files.resolve(files.join(dataDir, 'certs', 'rpc-auth-token.txt')),
```

Comment:
I see you want to store this in it's own file, versus using one of our existing config stores. Is there some reason why we don't use internal.json or config.json?

Reply:
Updated. store the token in internal.json